### PR TITLE
Fix failing tests and failing tests not marking exit code as 1

### PR DESCRIPTION
**Pull request by williamdes**
While working on packaging tcpdf into Debian I did see that failing tests still did exit 0, so I added some debugging and fixed the errors

- https://salsa.debian.org/phpmyadmin-team/tcpdf/-/jobs/1831241#L765
- https://github.com/tecnickcom/TCPDF/runs/3115139236#step:9:151

### CODE REVIEW

File: tests/launch.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ if [ $? -gt 0 ]; then
 fi
 
 # Only start here, the command checking can exit code > 0
-set -e
+set -eu
```
Comment (williamdes) [inline tests/launch.sh:12]: not related, adding a bash safety

File: tests/launch.sh [hunk 2]
```diff
@@ -96,7 +99,21 @@ for file in $EXAMPLE_FILES; do
             echo "Generated-invalid-file: $file"
         fi
     else
+        FAILED_FLAG=1
```
Comment (williamdes) [inline tests/launch.sh:102]: this one the line missing to mark the failure as a failure

**Comment by williamdes** [comment]
Yay !
It passes + coverage bump !

**Comment by williamdes** [comment]
Hi @nicolaasuni 
Would you mind merging some of the PRs please ?

**Comment by williamdes** [comment]
https://tracker.debian.org/pkg/tcpdf

Multi arch "tests" are passing  🎉 

Screenshot_2021-08-20-14-23-01-366_com brave browser

**Review by nicolaasuni — APPROVED** [review]

**Pull request merged**